package_manifest.yaml

```diff
@@ -501,6 +501,7 @@ foreman_nodejs_packages:
     nodejs-webpack:
       strategy: bundle
 
+    nodejs-yup: {}
```

Comment:
There is a strange blank line that shouldn't be there above this.

Reply:
Removed